CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -0,0 +1,173 @@
+# Release notes for v3.4.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v3.3.0
+
+## Changes by Kind
+
+### Feature
+
+- Add support for cross-namespace data sources alpha feature ([#805](https://github.com/kubernetes-csi/external-provisioner/pull/805), [@ttakahashi21](https://github.com/ttakahashi21))
+- Register metrics exposed by sig-storage-lib ([#792](https://github.com/kubernetes-csi/external-provisioner/pull/792), [@RaunakShah](https://github.com/RaunakShah))
+- Update the annotation that needs to be applied to VolumeSnapshotContents from snapshot.storage.kubernetes.io/allowVolumeModeChange to snapshot.storage.kubernetes.io/allow-volume-mode-change ([#791](https://github.com/kubernetes-csi/external-provisioner/pull/791), [@RaunakShah](https://github.com/RaunakShah))
+
+### Bug or Regression
+
+- Fix string pointer comparison for source volume mode conversion ([#793](https://github.com/kubernetes-csi/external-provisioner/pull/793), [@RaunakShah](https://github.com/RaunakShah))
+- Add defence logic for PV without ClaimRef ([#796](https://github.com/kubernetes-csi/external-provisioner/pull/796), [@zezaeoh](https://github.com/zezaeoh))
```

Comment:
I went ahead and updated the release notes on these PRs so that they can get automatically generated.

Reply:
Thanks! I re ran the script